client/client.go

```diff
@@ -37,22 +37,32 @@ type LogClient struct {
 	LogID    int64
 	client   trillian.TrillianLogClient
 	hasher   merkle.TreeHasher
-	STR      trillian.SignedLogRoot
-	MaxTries int
+	str      trillian.SignedLogRoot
+	maxTries int
 	pubKey   gocrypto.PublicKey
 }
 
 // New returns a new LogClient.
-func New(logID int64, client trillian.TrillianLogClient, hasher merkle.TreeHasher, pubKey gocrypto.PublicKey) *LogClient {
+func New(logID int64, client trillian.TrillianLogClient, hasher merkle.TreeHasher, pubKey gocrypto.PublicKey) LogVerifier {
 	return &LogClient{
 		LogID:    logID,
 		client:   client,
 		hasher:   hasher,
-		MaxTries: 3,
+		maxTries: 3,
 		pubKey:   pubKey,
 	}
 }
 
+// STR returns the current trusted SignedLogRoot
```

Comment:
Thanks for pointing this out. I'll make the comment a bit clearer. 
Is it standard practice for clients to blindly trust the first root they see? 
Is there a standard way of starting a client off from an empty root or something like that?